inc/Base/SubscriberSynchronization.php

```diff
@@ -68,6 +68,19 @@ public function smaily_checkout_subscribe_customer( $order_id ) {
 		$store = get_site_url();
 		$data['store'] = isset( $store ) ? $store : 'woocommerce';
 
+		// Language code if using WPML.
+		$lang = '';
+		if ( defined( 'ICL_LANGUAGE_CODE' ) ) {
+			$lang = ICL_LANGUAGE_CODE;
+			// Language code if using polylang.
+		} elseif ( function_exists( 'pll_current_language' ) ) {
+			$lang = pll_current_language();
+		} else {
+			$lang = get_locale();
```

Comment:
`ICL_LANGUAGE_CODE` and `get_locale` return different values. First returns `en`, but latter `en_US`. Ensure both provide similar value for `$data['language']`.

Reply:
Fixed in https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/27/commits/279bdd8f57579fcb459659b33854b32ac2f862ee